integrations/oci/tls-certificates/src/main/java/io/helidon/integrations/oci/tls/certificates/MetricsTracker.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.integrations.oci.tls.certificates;
+
+import io.helidon.inject.api.Contract;
+
+// consider making this more robust, public, and relocating this to somewhere under common or inject
+// it is here to track metrics (decoupled from mp)
+@Contract
+interface MetricsTracker {
```

Comment:
@tomas-langer I would like to discuss this, or follow-up with a tracking enhancement to handle metrics around oci-tls-manager.

Reply:
@tjquinno fyi.

Reply:
We will have a metric API that can be used even when metrics impl will not be available for such cases. No need to introduce it here